x/upgrade/types/storeloader_test.go

```diff
@@ -92,11 +94,12 @@ func TestSetLoader(t *testing.T) {
 			loadStoreKey: "foo",
 		},
 		"rename with inline opts": {
-			setLoader: useUpgradeLoader(0, &store.StoreUpgrades{
+			setLoader: useUpgradeLoader(upgradeHeight, &store.StoreUpgrades{
 				Renamed: []store.StoreRename{{
 					OldKey: "foo",
 					NewKey: "bar",
 				}},
+				Added: []string{"baz"},
```

Comment:
Is this addition used somewhere?

Reply:
I originally planned to add some tests for `Added` and `Deleted`. Can I add it via another PR?

Reply:
> Can I add it via another PR?

Sure! Could you create an issue to make sure it's tracked?